src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/prisoner/Prisoner.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import jakarta.validation.constraints.NotBlank
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Prisoner(
+  @NotBlank
+  val prisonerNumber: String,
```

Comment:
This is how we should be using it as well. However, according to Steve, the agreed name is PrisonNumber. We should have a chat with them.